.reuse/dep5

```diff
@@ -1,12 +1,12 @@
-# SPDX-FileCopyrightText: 2019 Serokell <https://serokell.io>
-# SPDX-License: Unlicense
+# SPDX-FileCopyrightText: 2020 Serokell <https://serokell.io>
+# SPDX-License: CC0-1.0
 
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 
 Files: templates/.github/*
-Copyright: 2019 Serokell <https://serokell.io>
-License: Unlicense
+Copyright: 2020 Serokell <https://serokell.io>
+License: CC0-1.0
 
 Files: templates/reuse/dep5
-Copyright: 2019 Serokell <https://serokell.io>
-License: Unlicense
+Copyright: 2020 Serokell <https://serokell.io>
```

Comment:
Isn't it more correct to change 2019 to 2019-2020 rather than just 2020?

Reply:
Yeah, you are right.